### PR TITLE
add an option to run in test-mode to skip DNS stuff

### PR DESCRIPTION
**Pull request by ddellacosta**
No PR for this--I found this annoying earlier today when testing and Jen also ran into issues before she found out she had to comment out a line to avoid DNS errors.  This is bad, so I'm adding an option to run this in a test import mode along with documenting it in the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,6 +73,30 @@ automate building and deploying to CoreOS.
...
+configuration, and the last argument is the CSV import file
+itself. This will add partner sites and set up the DNS as need be.
+
+If running in a local setup in order to simply test partner site
+creation, you probably don't want DNS configuration run--in that case,
+you can run the `test-import` variation to skip that:
+
+
+```bash
+$ lein run -m partner-works.import/test-import '{:host "localhost" :port 5672}' import-data.csv
```
Comment (7maples) [inline README.md]: I believe it should be port 45672

Comment (ddellacosta) [inline README.md]: Oh, I changed that on purpose--the goal here is to make it obvious this is RabbitMQ we're talking about, I'm assuming folks will change it as need be.

Comment (7maples) [inline README.md]: But wouldn't we want port 45672 100% of the time we're doing the test-import variation?

Comment (ddellacosta) [inline README.md]: Well, it depends on what your local RabbitMQ port is set up as--this is specific to the docker config.  And by the same logic I added `localhost` for the entry that will probably always be a production IP.

I don't feel super strongly about it, just trying to be consistent with the examples.  Maybe I should side-step the whole thing and use a placeholder value like `<your local RabbitMQ port>`?

Comment (ddellacosta) [inline README.md]: @7maples I tweaked it to be more descriptive--let me know if this is better!

**Comment by 7maples** [comment]
LGTM

**Pull request merged**